torch/_inductor/kernel/mm_plus_mm.py

```diff
@@ -26,6 +27,9 @@
     M = {{size("A", 0)}}
     N = {{size("B", 1)}}
     K1 = {{size("A", 1)}}
+    if M * N * K1 == 0:
```

Comment:
this change isn't being tested because of the other changes.. what happens when you remove the other changes, and just include this ?

Reply:
I've tried this. The generated Triton kernels are running fine and take exactly the same time as the aten one.